ci/infrastructure/scripts/infrastructure.sh

```
@@ -855,7 +858,7 @@ packageSSRArtifact() {
 # create Docker container
 containerCreateAndStart() {
   if [ "$VS_CONTAINER_PRESERVE" == "TRUE" ] && [ -z "$CONTAINER_ID" ]; then
-    echo "$(eval $VS_LOG_DATESTAMP) INFO  [$VS_SCRIPTNAME] VS_CONTAINER_PRESERVE was set to $VS_CONTAINER_PRESERVE, but no container was found, starting new contanier"
+    echo "$(eval $VS_LOG_DATESTAMP) INFO  [$VS_SCRIPTNAME] VS_CONTAINER_PRESERVE was set to $VS_CONTAINER_PRESERVE, but no container was found, starting new container"
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)